### PR TITLE
Add IceNet Notebook

### PR DESCRIPTION
**Pull request by acocac**
The notebook demonstrates IceNet using sample data of climate simulations and observational data fetched from a Zenodo repository.

### CODE REVIEW

**Comment by acocac** [comment]
@nbarlowATI the notebook is ready for revision. Thanks for helping me to sort out the issue with the conda environment!

File: IceNet/polar-modelling-icenet.ipynb [hunk 1]
```diff
@@ -0,0 +1,1411 @@
+{
```
Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>"ground thruth" =&gt; "ground truth"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>This path will depend on where the notebook is being run - I guess it makes sense to leave the one here as the one that works on the AzureML VM, but maybe could add a note to the text to point that out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>"same input data of Nature Communications' paper " =&gt; "same input data as in the  Nature Communications paper "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>"data as the forecast predictions" =&gt; "data, such as the forecast predictions"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>Maybe I missed it, but is there an explanation somewhere of where the ground truth SIC data comes from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>"use a sample data" =&gt; "use a small sample of the data"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>Very nice!! This interactive plot is great!!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Comment (nbarlowATI) [inline IceNet/polar-modelling-icenet.ipynb:1]: <p>For this plot, and the next one, when I ran them, the axis labels were cut off a little bit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>done!</p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>done!</p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>done!</p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>done, thanks!</p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>which browser have you used? I remember the axis labels issue in Safari. </p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>Thanks, Tom and I tried one with <code>hvplot</code> but it wasn't giving the most aesthetic results. We ended using <code>panels</code> which is a great python library for generating dashboard and support <code>Matplotlib</code> plots.  </p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>I'm adding the following text to the <strong>Download ground truth SIC </strong>section explaining further details of the the ground truth SIC data.</p><p> </p><blockquote>We additionally download analysis-ready ground truth SIC data from a zenodo repository.</blockquote><blockquote><br></blockquote><blockquote>:::{note}</blockquote><blockquote>The analysis-ready ground truth SIC data were generated by running the script <code>python3 icenet/download_sic_data.py</code> in step **2) Download data** according to the [icenet-paper repository](https://github.com/tom-andersson/icenet-paper). The script downloads and concatenate [OSI-SAF SIC data](https://osisaf-hl.met.no/v2p1-sea-ice-index), OSI-450 (1979-2015) and OSI-430-b (2016-ownards), and saves it as monthly averages in a <code>netCDF</code> file.</blockquote><blockquote>:::</blockquote><p> </p>

Comment (acocac) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>I'll add a the following text:</p><p> </p><blockquote>Note the python route after the exclamation in the line below works on the AzureML VM. The route might change in your local system. </blockquote><p> </p><p>How does it sound?</p>

Comment (scotthosking) [reply IceNet/polar-modelling-icenet.ipynb:1]: <p>Thanks Nick!</p>

**Pull request merged**